roles/setup_pgbackrest/tasks/exchange_ssh_keys.yml

```diff
@@ -72,7 +72,6 @@
   ansible.builtin.command: ssh-keyscan {{ _pg_host }}
   register: _pgbackrest_ssh_keyscan_output
   delegate_to: "{{ _pgbackrest_server_inventory_hostname }}"
-  delegate_facts: true
```

Comment:
Is there a specific reason for removing the delegate_facts? I hope its removal won't have a negative impact.

Reply:
It was causing the `ssh-keyscan` command to only collect and register information from one of the primary/standby nodes, not both. This meant that (in the test case), only the primary information was included in the `known_hosts` file on the pgbackrestserver node, which was causing authentication errors in the validation checks when the server node was trying to connect to the primary and standby. Removing solved this issues and the `known_hosts` file was updated properly.

Reply:
Please update the README to include information about the new variable, standby_present or any other changes